spark/v3.4/spark/src/main/java/org/apache/iceberg/spark/source/SparkPositionDeletesRewrite.java

```diff
@@ -56,7 +56,7 @@
 
 /**
  * {@link Write} class for rewriting position delete files from Spark. Responsible for creating
- * {@link PositionDeleteBatchWrite}.
+ * PositionDeleteBatchWrite.
```

Comment:
Fixing javadoc link warnings (seems it cannot link to inner class javaDocs)

Reply:
What if you include the parent class?

Reply:
Ah right, fixed.